lib/dialects/mysql/schema/mysql-tablecompiler.js

```diff
@@ -135,6 +141,20 @@ class TableCompiler_MySQL extends TableCompiler {
     });
   }
 
+  primaryKeys() {
+    const pks = filter(this.grouped.alterTable || [], { method: 'primary' });
```

Comment:
why use lodash filter instead of a native one?

Reply:
No reason! I originally based it off the sqlite3 tablecompiler, which also uses the lodash filter

Reply:
I can clean this up once I've fixed the tests